File: WebLeave/API/Dtos/SeaHr/PermissionRightsDTO.cs
namespace API.Dtos.SeaHr
{
    public class PermissionRightsDTO
    {
        public int STT { get; set; }
        public string EmpNumber { get; set; }
        public string EmpName { get; set; }
        public string Part { get; set; }
        public string PositionName { get; set; }
        public string ApprovalUsers { get; set; }
    }

    public class PermissionParam
    {
        public string EmpNumber { get; set; }
        public int PartID { get; set; }
        // title excel
        public string Label_Stt { get; set; }
        public string Label_EmpNumber { get; set; }
        public string Label_EmpName { get; set; }
        public string Label_PositionName { get; set; }
        public string Label_Part { get; set; }
        public string Label_ApprovalUsers { get; set; }
    }
}

File: WebLeave/SPA/src/app/_core/models/seahr/permissionRightsDTO.ts
export interface PermissionRightsDTO {
    stt: number;
    empNumber: string;
    empName: string;
    positionName: string;
    part: string;
    approvalUsers: string;
}

export interface PermissionParam {
    empNumber: string;
    partID: number;
    // title excel
    label_Stt: string;
    label_EmpNumber: string;
    label_EmpName: string;
    label_PositionName: string;
    label_Part: string;
    label_ApprovalUsers: string;

}
